addons/csi-azurefile/ccm-azurefile.yaml

```diff
@@ -657,3 +703,923 @@ status:
+            - "--allow-inline-volume-key-access-with-identity=false"
+          ports:
+            - containerPort: 29613
+              name: healthz
+              protocol: TCP
+          livenessProbe:
+            failureThreshold: 5
+            httpGet:
+              path: /healthz
+              port: healthz
+            initialDelaySeconds: 30
+            timeoutSeconds: 10
+            periodSeconds: 30
+          env:
+            - name: AZURE_CREDENTIAL_FILE
+              valueFrom:
+                configMapKeyRef:
+                  name: azure-cred-file
+                  key: path
+                  optional: true
+            - name: CSI_ENDPOINT
+              value: unix:///csi/csi.sock
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+            - name: AZURE_GO_SDK_LOG_LEVEL
+              value:
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - mountPath: /csi
+              name: socket-dir
+            - mountPath: /var/lib/kubelet/
+              mountPropagation: Bidirectional
+              name: mountpoint-dir
+            - mountPath: /etc/kubernetes/
+              name: azure-cred
+            - mountPath: /dev
+              name: device-dir
+          resources:
+            limits:
+              memory: 400Mi
+            requests:
+              cpu: 10m
+              memory: 20Mi
+      volumes:
+        - hostPath:
+            path: /var/lib/kubelet/plugins/file.csi.azure.com
+            type: DirectoryOrCreate
+          name: socket-dir
+        - hostPath:
+            path: /var/lib/kubelet/
+            type: DirectoryOrCreate
+          name: mountpoint-dir
+        - hostPath:
+            path: /var/lib/kubelet/plugins_registry/
+            type: DirectoryOrCreate
+          name: registration-dir
+        - hostPath:
+            path: /etc/kubernetes/
+            type: DirectoryOrCreate
+          name: azure-cred
+        - hostPath:
+            path: /dev
+            type: Directory
+          name: device-dir
+---
+# Source: azurefile-csi-driver/templates/csi-azurefile-controller.yaml
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csi-azurefile-controllermanager
+  namespace: kube-system
+  labels:
+    app: csi-azurefile-controllermanager
+    app.kubernetes.io/name: azurefile-csi-driver
+    app.kubernetes.io/instance: release-name
+    app.kubernetes.io/component: csi-driver
+    app.kubernetes.io/part-of: azurefile-csi-driver
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/version: "v1.27.1"
+    helm.sh/chart: azurefile-csi-driver-v1.27.1
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: azurefile-csi-driver
+      app.kubernetes.io/instance: release-name
+      app: csi-azurefile-controllermanager
```

Comment:
This requires migration.

Reply:
No, because I've override the name in helm values, so it's a new resource.